inc/app_replacer.php

```diff
@@ -368,8 +368,13 @@ public function set_properties() {
 		];
 		$this->upload_resource['url_length'] = strlen( $this->upload_resource['url'] );
 
-		$content_parts = parse_url( content_url() );
+		$site_url = get_site_url();
+		$content_url = content_url();
 
+		if ( strpos( $content_url, $site_url ) === false && strpos( $content_url, '/' ) === 0 ) {
```

Comment:
Isn't this condition `strpos( $content_url, '/' ) === 0` enough?

Reply:
yes it is, changed it